README.md

```diff
@@ -1,26 +1,143 @@
 
-To get a unit test coverage report, you can run `sbt clean coverage test coverageReport`,
-then open the resulting coverage report `target/scala-2.12/scoverage-report/index.html` in a web browser.
+## Running the service
 
-Test coverage threshold is set at 80% - so if you commit any significant amount of implementation code without writing tests, you can expect the build to fail.
+*From the root directory*
+
+`sbt run` - starts the service locally
+
+`sbt runAllChecks` - Will run all checks required for a successful build
+
+Default service port on local - 9395
+
+### Required dependencies
+
+There are a number of dependencies required to run the service.
+
+The easiest way to get started with these is via the service manager CLI - you can find the installation guide [here](https://docs.tax.service.gov.uk/mdtp-handbook/documentation/developer-set-up/set-up-service-manager.html)
+
+| Command                                          | Description |
+| --------                                         | ------- |
+| `sm2 --start CUSTOMS_FINANCIALS_ALL`             | Runs all dependencies |
+| `sm2 -s`                                         | Shows running services |
+| `sm2 --stop CUSTOMS_GUARENTEE_ACCOUNT_FRONTEND` | Stop the micro service  |
+| `sbt run`                                        | (from root dir) to compile the current service with your changes |
+
+
+### Runtime Dependencies
+
+* `AUTH`
+* `AUTH_LOGIN_STUB`
+* `AUTH_LOGIN_API`
+* `BAS_GATEWAY`
+* `CA_FRONTEND`
+* `SSO`
+* `USER_DETAILS`
+* `CUSTOMS_FINANCIALS_API`
+* `CUSTOMS_FINANCIALS_HODS_STUB`
+* `CUSTOMS_FINANCIALS_SDES_STUB`
+* `CONTACT_FRONTEND`
+
+### Login enrolments
+
+The service can be accessed by using below enrolments and with below sample EORI numbers, via http://localhost:9949/auth-login-stub/gg-sign-in (on local) or https://<host:port>/auth-login-stub/gg-sign-in on DEV/QA/STAGING
+
+Redirect URL - `/customs/payment-records`
+
+| Enrolment Key	| Identifier Name | Identifier Value |
+| -------- | ------- | ------- |
+| `HMRC-CUS-ORG` | `EORINumber`| `GB744638982000` |
+| `HMRC-CUS-ORG` | `EORINumber`| `GB744638982001` |
+
+## Testing
+
+The minimum requirement for test coverage is 90%. Builds will fail when the project drops below this threshold.
+
+### Unit Tests
+
+| Command    | Description |
+| -------- | ------- |
+| `sbt test` | Runs unit tests locally |
+| `sbt "test:testOnly *TEST_FILE_NAME*"` | runs tests for a single file |
+
+### Coverage
+
+| Command    | Description |
+| -------- | ------- |
+| `sbt clean coverage test coverageReport` | Generates a unit test coverage report that you can find here target/scala-3.3.5/scoverage-report/index.html  |
+
+## Available Routes
+
+You can find a list of microservice specific routes here - `/conf/app.routes`
+
+Application entrypoint:  `/customs/payment-records` 
+
+| Path                            |
```

Comment:
I personally feel like we should not bother listing any routes here and just point to `app.routes`.

I think this is only necessary for `customs-data-store` and potentially `customs-financials-api` to assist other teams.